src/help.txt

```diff
@@ -0,0 +1,9 @@
+usage: jotdown [option] <file>
```

Comment:
should be `[file]` here, right? as it is an optional argument, currently

Reply:
That is correct, my bad!